docs-website-src/content/blockers.md

```diff
@@ -84,8 +100,12 @@ Once ELevate has completed, you should then perform the update to the PostgreSQL
 
 ## Using OVH proactive intervention monitoring
 
-If you are using a dedicated server hosted at OVH, you should **disable the `proactive monitoring` before starting** the elevation process.
+If you are using a dedicated server hosted at OVH, you should **disable the `proactive monitoring` before starting** the elevation process.  To indicate you have done this, you must create the touch file `/var/cpanel/acknowledge_ovh_monitoring_for_elevate` or elevation will block when it detects that the system is hosted by OVH.
 The proactive monitoring incorrectly detects an issue on your server during one of the reboots.
 Your server would then boot to a rescue mode, interrupting the elevation upgrade.
 
 [Read more about OVH monitoring](https://support.us.ovhcloud.com/hc/en-us/articles/115001821044-Overview-of-OVHcloud-Monitoring-on-Dedicated-Servers)
+
+# Leapp preupgrade (dry run) check
+
+If no issues are found, Elevate will perform one more check before performing the upgrade: it will perform a "dry run" of the leapp upgrade by executing `leapp preupgrade`.  This will point out any problems that leapp would encounter during the actual upgrade.  If any errors are found, they will need to be addressed before performing the upgrade.  This test is only performed when the script is invoked with the --start option.
```

Comment:
```suggestion
If no issues are found, Elevate will perform one more check before performing the upgrade: it will perform a "dry run" of the Leapp upgrade by executing `leapp preupgrade`.  This will point out any problems that Leapp would encounter during the actual upgrade.  If any errors are found, they will need to be addressed before performing the upgrade.  This test is only performed when the script is invoked with the --start option.
```